scylladb/scylladb_test.go

```diff
@@ -0,0 +1,114 @@
+package scylladb
+
+import (
+	"github.com/gofiber/utils"
```

Comment:
All the storage tests should use testify for Asserts. See here https://github.com/gofiber/storage/blob/main/redis/redis_test.go